cpu/stm32_common/periph/uart.c

```diff
@@ -117,24 +117,21 @@ int uart_init(uart_t uart, uint32_t baudrate, uart_rx_cb_t rx_cb, void *arg)
     return UART_OK;
 }
 
-void uart_write(uart_t uart, const uint8_t *data, size_t len)
+static void send_byte(uart_t uart, uint8_t byte)
 {
-    assert(uart < UART_NUMOF);
-
-    for (size_t i = 0; i < len; i++) {
 #if defined(CPU_FAM_STM32F0) || defined(CPU_FAM_STM32L0) \
     || defined(CPU_FAM_STM32F3) || defined(CPU_FAM_STM32L4) \
     || defined(CPU_FAM_STM32F7)
         while (!(dev(uart)->ISR & USART_ISR_TXE)) {}
-        dev(uart)->TDR = data[i];
+        dev(uart)->TDR = byte;
```

Comment:
This line looks over-indented. It should be 4 spaces shifted. Same for the lines above and below.

Reply:
yup, fixed. I also inlined this function and `wait_for_tx_complete`